package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "sx-monorepo",
+  "license": "MIT",
```

Comment:
This shouldn't really matter as we don't publish root package (it's private).

Reply:
It's just so it show up on GitHub

Reply:
It reads it from LICENSE/LICENDE.md file I belive.

Reply:
Right, then maybe it can help if someone open this file